modules/admin/con-rbac-rest-api.adoc

```diff
@@ -11,28 +11,39 @@ Using the RBAC REST API, you can perform the following actions:
 
 The RBAC REST API requires the following components:
 
-*Authorization*
+Authorization::
++
+--
 
 The RBAC REST API requires Bearer token authorization for the permitted user role. For development purposes, you can access a web console in a browser. When you refresh a token request in the list of network requests, you find the token in the response JSON.
 
 `Authorization: Bearer $token`
 
-For example, on the *Homepage* of the {product-short}, you can navigate to the *Network* tab and search for the `query?term=` network call. Alternatively, you can go to the *Catalog* page and select any network call with `entity-facets` to acquire the Bearer token. 
+For example, on the *Homepage* of the {product-short}, you can navigate to the *Network* tab and search for the `query?term=` network call. Alternatively, you can go to the *Catalog* page and select any Catalog API network call to acquire the Bearer token.
```

Comment:
```suggestion
For example, on the {product-short} *Homepage*, navigate to the *Network* tab and search for the `query?term=` network call. Alternatively, you can go to the *Catalog* page and select any Catalog API network call to acquire the Bearer token.
```

Reply:
This is a procedure and should be formatted as such. Let's create some Jira tickets to reformat these con modules into proc modules in the future.